### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by SyedTauhidUllahShah** [cheungdaven/metricfactorization#7]
Kindly explain this
self.loss = tf.reduce_sum((1 + 0.1 * self.y) * tf.square(
            (self.y * self.pred + (1 - self.y) * tf.nn.relu(self.beta * (1 - self.y) - self.pred))))
This is different from what u have written in the paper.

**Comment by cheungdaven** [comment]
I have revised the loss. 
self.loss = tf.reduce_sum((1 +  0.1 * self.y ) * tf.square(self.beta * (self.one - self.y) - self.pred_y))

**Comment by SyedTauhidUllahShah** [comment]
What's does self. one mean?

**Comment by cheungdaven** [comment]
oh, sorry about that, it should be 1.
